app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -275,7 +282,11 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
                         onCannotOpenFile = {
                             FenixSnackbar.make(view, Snackbar.LENGTH_SHORT)
                                 .setText(context.getString(R.string.mozac_feature_downloads_could_not_open_file))
-                                .setAnchorView(browserToolbarView.view)
+                                .setAnchorView(
+                                    if (view.context.settings().shouldUseBottomToolbar)
+                                        browserToolbarView.view
+                                    else null
```

Comment:
I'm not clear on what null does to the behavior here.  Would you mind adding a short comment explaining what that does?  (also easier if this check is pulled into a method)